src/Components/Patient/FileUpload.tsx

```diff
@@ -583,11 +585,11 @@ export const FileUpload = (props: FileUploadProps) => {
                                   setEditFileName(item?.name);
                                   setModalOpenForEdit(true);
                                 }}
-                                className="btn btn-primary m-1 sm:w-auto w-full"
+                                className="m-1 sm:w-auto w-full"
                               >
                                 <i className="fa-solid fa-pencil mr-2"></i>EDIT
                                 FILE NAME
```

Comment:
ButtonV2 already applies `flex gap-2`, so `mr-2` will add unnecessary more space.
Also use `CareIcon`
```suggestion
                                <i className="fa-solid fa-pencil" />
                                <span>EDIT FILE NAME</span>
```